pypesto/optimize/optimizer.py

```diff
@@ -535,7 +538,7 @@ def __init__(self, options: Dict = None):
         super().__init__()
 
         if options is None:
-            options = {'maxiter': 200}
+            options = {'maxiter': 10000}
```

Comment:
please do not change the default options here

Reply:
Done